### PR TITLE
PAN-268 Add classification filter

### PR DESCRIPTION
**Pull request by PawelPodkalicki**

### CODE REVIEW

File: app/Models/Classification.php [hunk 1]
```diff
@@ -40,6 +40,10 @@ class Classification extends Model
     use Ownership;
     use AutomateMutators;
 
+    public const DB_STATUS_REJECTED = 0;
```
Comment (teghnet) [inline app/Models/Classification.php]: Loose the `DB_` prefix

File: app/Models/NetworkBanner.php [hunk 2]
```diff
@@ -36,6 +39,12 @@
  */
 class NetworkBanner extends Model
 {
+    public const HTML_TYPE = 'html';
```
Comment (teghnet) [inline app/Models/NetworkBanner.php]: `TYPE_HTML` is more readable when we have more types

**Review by teghnet — DISMISSED** [review]

File: app/Models/Classification.php (hunk 1 above)
Comment (PawelPodkalicki) [reply app/Models/Classification.php]: changed

File: app/Models/NetworkBanner.php (hunk 2 above)
Comment (PawelPodkalicki) [reply app/Models/NetworkBanner.php]: changed

**Review by teghnet — APPROVED** [review]

**Review by c3zi — APPROVED** [review]

**Pull request merged**